ccdproc/tests/test_image_collection.py

```diff
@@ -657,6 +657,20 @@ def test_sorting(self, triage_setup):
         for i in range(len(collection.summary)):
             assert(collection.summary['file'][i] == collection.files[i])
 
+    def test_duplicate_keywords(self, triage_setup):
+        # Make sure duplicated keywords don't make the imagefilecollection
+        # fail.
+        hdu = fits.PrimaryHDU()
+        hdu.data = np.zeros((5, 5))
+        hdu.header['stupid'] = 'fun'
+        hdu.header.append(('stupid', 'nofun'))
+
+        hdu.writeto(os.path.join(triage_setup.test_dir, 'duplicated.fits'))
+
+        ic = image_collection.ImageFileCollection(triage_setup.test_dir,
+                                                  keywords='*')
+        assert 'stupid' in ic.summary.colnames
```

Comment:
Sounds good 👍. I added the test and one for `assert 'fun' in ic.summary['stupid']`.